libs/fluxc-plugin/src/test/java/org/wordpress/android/fluxc/store/WCGlobalAttributeStoreTest.kt

```diff
@@ -1,50 +1,35 @@
 package org.wordpress.android.fluxc.store
 
-import com.yarolegovich.wellsql.WellSql
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Before
 import org.junit.Test
-import org.junit.runner.RunWith
 import org.mockito.kotlin.mock
-import org.mockito.kotlin.spy
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
-import org.robolectric.RobolectricTestRunner
-import org.robolectric.RuntimeEnvironment
-import org.robolectric.annotation.Config
-import org.wordpress.android.fluxc.SingleStoreWellSqlConfigForTests
-import org.wordpress.android.fluxc.model.SiteModel
 import org.wordpress.android.fluxc.model.attribute.WCGlobalAttributeMapper
 import org.wordpress.android.fluxc.network.rest.wpcom.wc.WooPayload
 import org.wordpress.android.fluxc.network.rest.wpcom.wc.product.attributes.ProductAttributeRestClient
-import org.wordpress.android.fluxc.persistence.WellSqlConfig
 import org.wordpress.android.fluxc.persistence.dao.GlobalAttributesDao
 import org.wordpress.android.fluxc.test
 import org.wordpress.android.fluxc.tools.initCoroutineEngine
 import org.wordpress.android.fluxc.wc.attributes.WCProductAttributesTestFixtures.attributesFullListResponse
 import org.wordpress.android.fluxc.wc.attributes.WCProductAttributesTestFixtures.parsedAttributesList
 import org.wordpress.android.fluxc.wc.attributes.WCProductAttributesTestFixtures.stubSite
 
-@Config(manifest = Config.NONE)
-@RunWith(RobolectricTestRunner::class)
 class WCGlobalAttributeStoreTest {
     private lateinit var storeUnderTest: WCGlobalAttributeStore
-    private lateinit var restClient: ProductAttributeRestClient
-    private lateinit var globalAttributesDao: GlobalAttributesDao
-    private lateinit var mapper: WCGlobalAttributeMapper
+    private val restClient = mock<ProductAttributeRestClient>()
+    private val globalAttributesDao = mock<GlobalAttributesDao>()
+    private val mapper = mock<WCGlobalAttributeMapper>()
```

Comment:
I know you didn't introduce this, but please let's not mock mappers

```suggestion
    private val mapper = WCGlobalAttributeMapper()
```

Reply:
FYI: This is now done: 51df2f51961259007249a4eb2f8a490370f83bf7 @wzieba